test/AutoRest.TestServer.Tests/Infrastructure/JsonAsserts.cs

```diff
@@ -0,0 +1,24 @@
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Azure.Core;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests.Infrastructure
+{
+    internal static class JsonAsserts
+    {
+        public static void AssertSerialization(string expected, IUtf8JsonSerializable serializable)
+        {
+            var memoryStream = new MemoryStream();
```

Comment:
IIRC, it doesn't actually do much for a `MemoryStream`, but should use the `using` keyword here as well (could use new one without new block).